webdriver/tests/bidi/input/perform_actions/__init__.py

```diff
@@ -1,17 +1,9 @@
 from webdriver.bidi.modules.script import ContextTarget
-
-
-def remote_mapping_to_dict(js_object):
-    obj = {}
-    for key, value in js_object:
-        obj[key] = value["value"]
-
-    return obj
-
+from ... import get_viewport_dimensions, remote_mapping_to_dict
```

Comment:
Please separate 3rd party and local imports.

```suggestion

from ... import get_viewport_dimensions, remote_mapping_to_dict
```